androidsdk/src/test/java/com/wootric/androidsdk/SurveyValidatorTest.java

```diff
@@ -63,6 +63,9 @@ public void checksEligibility_whenNotRecentlySurveyed() throws Exception {
     public void doesNotCheckEligibility_whenRecentlySurveyed() throws Exception {
         User user = testUser();
         EndUser endUser = testEndUser();
+        long createdAtJustNow = new Date().getTime();
```

Comment:
@hachibu so I've checked this more closely. There is this method
`public void setEndUserCreatedAt(long createdAt) {

        if (createdAt > 9999999999L){
            Log.w(TAG, "WARNING: The created date exceeds the maximum 10 characters allowed. " +
                    "If you are using System.currentTimeMillis() divide it by 1000.");
        }
        endUser.setCreatedAt(createdAt);
    }`

But when this is set, it affects the `external_created_at` field.
All date comparisons in the SDK are done in milliseconds. So in the case for this test, this is correct as the validation needs to be done in milliseconds.